CRM/Utils/Money.php

```diff
@@ -260,14 +264,14 @@ public static function formatLocaleNumericRoundedByPrecision($amount, $precision
    *
    * @param string $amount
    * @param int $precision
+   * @param string $locale
    *
    * @return string
    *   Formatted amount.
    */
-  public static function formatLocaleNumericRoundedByOptionalPrecision($amount, $precision) {
+  public static function formatLocaleNumericRoundedByOptionalPrecision($amount, $precision, $locale = NULL) {
```

Comment:
@eileenmcnaughton I am not sure what the difference between this and formatLocaleNumericRoundedByPrecision is meant to be now?

Reply:
That implies there are some of these functions you CAN still track....

Reply:
@seamuslee001 so the idea when we set up these functions was to have meaningful function names instead of functions with endless variables - we seem to have failed on both counts!

However, I guess there are some variables that we have in combo

1) show currency or not
2) use default locale or machine locale or provided locale
3) use default separators or currency+locale relevant processors
4) no rounding vs currency-relevant rounding vs provided rounding

does that cover all the things we are trying to account for?